src/registrar/templates/admin/change_list_results.html

```diff
@@ -13,57 +13,49 @@
 {% endif %}
 {% if results %}
 <div class="results override-change_list_results">
-<table id="result_list" class="usa-table usa-table--borderless usa-table--striped">
+<table id="result_list">
```

Comment:
**question (non-blocking)**: I don't understand why we would have to remove the USWDS table styling classes from this element in order to address this bug?

Reply:
Thank you for catching that, I'll revert (I missed this while breaking away code)